src/main/java/org/sonatype/repository/conan/internal/proxy/ConanProxyFacet.java

```diff
@@ -166,12 +196,12 @@ private Content putMetadata(final String assetPath,
           attributesMap = new AttributesMap();
           break;
       }
-      return doSaveMetadata(assetPath, tempBlob, content, assetKind, attributesMap, project, version, group);
+      return doSaveMetadata(assetPath, tempBlob, content, assetKind, attributesMap, coords);
     }
   }
 
-  private String getProjectIndexName(final String group, final String project, final String version) {
-    return group + "/" + project + "/" + version + "/index.json";
+  private String getProjectIndexName(final ConanCoords coords) {
+    return coords.getGroup() + "/" + coords.getProject() + "/" + coords.getVersion() + "/index.json";
```

Comment:
SUGGESTION/TRIVIAL: Might be worth moving this into a Utils helper like we do with other path stuff?

Reply:
Or maybe add a method to ConanCoords that returns it as a String?

Reply:
Perfect!